Updated Analytics API docs and added release note

suyashjoshi opened the pull request.

byrnereese commented on docs/analytics/aggregate.md:
```diff
@@ -1,15 +1,20 @@
+# Call Aggerate Reports
 
-## Composing a request to the Call Performance Aggregate API
+It is quite common to acquire and segment analytics data. Call Performance Aggregate API provides ability to aggregate call realted data in various ways such as by summing, averaging, getting minimum or maximum or percetage of data etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose API Reques ...
```
Please review this content for grammar and spelling. Install Grammarly or some other checker to ensure that sentence structure makes sense, and all words are spelled correctly.

suyashjoshi replied:
ok

byrnereese commented on docs/analytics/aggregate.md:
```diff
@@ -1,15 +1,20 @@
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/aggregatePerformanceReportCalls) for details about Aggregate API Endpoint.
+## Composing a request using RingCentral API Reference
+
+One can use our [interactive API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/aggregatePerformanceReportCalls) to not only create API Request but also run it interactively and see the response.
```
Please review for grammar

suyashjoshi replied:
ok

byrnereese commented on docs/analytics/timeline.md:
```diff
@@ -1,15 +1,20 @@
+It is quite common to acquire and segment analytics data based on time realted metrics. Call Performance Timeline API provides ability to segment call realted data in various ways such as by getting call duration based on response or actions etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose  ...
 
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) for details about Timeline API Endpoint.
+## Composing a request using RingCentral API Reference
```
Check for grammar. "Composing a request using the RingCentral..."

byrnereese commented on docs/analytics/timeline.md:
```diff
@@ -1,15 +1,20 @@
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) for details about Timeline API Endpoint.
+## Composing a request using RingCentral API Reference
+
+One can use our [interactive API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) to not only create API Request but also run it interactively and see the response.
```
Grammar: to not only create **an** API request, but also run it interactively to see the response.

suyashjoshi replied:
fixed

byrnereese commented on docs/basics/changelog/1_0_51.md:
```diff
@@ -0,0 +1,87 @@
+# API Changelog for Apr 2022 (v.1.0.51)
+
+### Voice
+
+### Analytics
+
```
Do we need to keep these, or can they be removed?

suyashjoshi replied:
removed 'Analytics' as it's also in subheading but kept 'Voice' as it's top-level category

suyashjoshi commented:
Btw @CoCraig I've also incorporated your changes and sending new PR today.

suyashjoshi commented:
Here is the link to review the changes (just merged them), I've incorporated all your feedback and fixed the page with 'grammar' issues - ran through grammarly, added key uses that @CoCraig asked so both API pages are well understood and also fixed formatting issues etc . https://github.com/ringcentral/ringcentral-api-docs/commit/89f2a9813a33d6c9ace790e38b468d533c8923f5
